test/unit/utils/utils.normalize.js

```diff
@@ -33,4 +33,63 @@ describe("Normalize utility", function() {
 
   });
 
-});
\ No newline at end of file
+  describe(".expandPK()", function() {
+    it("casts integers", function() {
+      var context = {
+        attributes: {
+          id: {
+            type: 'integer',
+            primaryKey: true
+          }
+        }
+      };
+
+      var options = {
+        id: '123'
+      }
+
+      var result = normalize.expandPK(context, options);
+
+      assert(result.id === 123);
+    });
+
+    it("casts uuids", function() {
```

Comment:
can we add a test here if you try to pass like `'foo'` or `7` or `null` for the value?